test/integration/cluster/cluster_helpers.py

```diff
@@ -0,0 +1,88 @@
+import os
+import shutil
+import subprocess
+
+import etcd3
+import yaml
+
+etcd_username = "root"
+etcd_password = "password"
+
+
+class EtcdInstance():
```

Comment:
I see the code into to places: test/integration/cluster and test/integration/replicaset. It could be moved in a some common place.

Reply:
> Could we move the code into `test/utils.py` or `test/etcd_helper.py`? @psergee, what do you think?

I like to idea to move etcd specific code to a separate helpers file.

Reply:
I moved etcd instance code to the test/etcd_helper.py.